module/applications/components/inventory.mjs

```diff
@@ -626,7 +626,7 @@ export default class InventoryElement extends HTMLElement {
    * @protected
    */
   _onRollRecharge(entry, { event }={}) {
-    if ( entry instanceof Item5e ) return item.system.uses?.rollRecharge({ apply: true, event });
+    if ( entry instanceof Item5e ) return entry.system?.uses?.rollRecharge({ apply: true, event });
```

Comment:
If `entry` is an instance of `Item5e`, we can be certain it has a `system` property.

```suggestion
    if ( entry instanceof Item5e ) return entry.system.uses?.rollRecharge({ apply: true, event });
```